test failure: trx_finality_status_if_test

https://github.com/AntelopeIO/spring/actions/runs/10112239885/job/27966354527?pr=414

```
Traceback (most recent call last):
  File "/__w/spring/spring/build/tests/trx_finality_status_test.py", line 201, in <module>
    assert state == unknownState, \
AssertionError: ERROR: Calling getTransactionStatus after the success_duration should have resulted in an "IRREVERSIBLE" state.
status: {
 "state": "IRREVERSIBLE",
 "block_number": 167,
 "block_id": "000000a7eb9a71a7a06577efbbf3d6f873d6b9ea64948d03557673b34872e13a",
 "block_timestamp": "2024-07-26T13:54:01.000",
 "expiration": "2024-07-26T13:55:30.000",
 "head_number": 286,
 "head_id": "0000011ebbbef4fe1a85f348613b6b44e8c35bfca2289fc08fda1d82ce6fb98c",
 "head_timestamp": "2024-07-26T13:55:00.500",
 "irreversible_number": 284,
 "irreversible_id": "0000011c66e069264dfc504c6cc411da5f2e91bb0783182767c0f3f19d23efda",
 "irreversible_timestamp": "2024-07-26T13:54:59.500",
 "earliest_tracked_block_id": "00000021b7d8f2f387e86ab61d2b2051e74fd19479a8b84f6d9bf70f70bd4795",
 "earliest_tracked_block_number": 33
}
```

--- a/tests/trx_finality_status_test.py
+++ b/tests/trx_finality_status_test.py
@@ -192,14 +192,15 @@ try:
         f"\n\nfinal status: {json.dumps(retStatus, indent=1)}"
     validate(retStatus)
 
+    # The transaction status db is purged on timeouts when blocks are processed, leeway allows small delay in blocks
     leeway=4
-    assert testNode.waitForBlock(blockNum=startingBlockNum+(successDuration*2),timeout=successDuration+leeway)
+    assert testNode.waitForBlock(blockNum=startingBlockNum+(successDuration*2)+leeway,timeout=successDuration+leeway)
 
     recentBlockNum=testNode.getBlockNum()
     retStatus=testNode.getTransactionStatus(transId)
     state = getState(retStatus)
     assert state == unknownState, \
-        f"ERROR: Calling getTransactionStatus after the success_duration should have resulted in an \"{irreversibleState}\" state.\nstatus: {json.dumps(retStatus, indent=1)}"
+        f"ERROR: Calling getTransactionStatus after the success_duration should have resulted in an \"{unknownState}\" state.\nstatus: {json.dumps(retStatus, indent=1)}"
     validateTrxState(retStatus, present=False)
     validate(retStatus, knownTrx=False)
     assert recentBlockNum <= retStatus["head_number"], \

